product_docs/docs/edbcloud/beta/administering_cluster/01a_superset_access.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Managing Superset Access
+---
+
+You control access to Superset data sources by Superset roles and permissions. Learn more about Superset roles, users, and permissions management in [Superset Security](https://superset.apache.org/docs/security). To assign Superset permissions to the EDB Cloud user role, see [Changing Role Permissions](01_portal_access/#changing-role-permissions).
+
+EDB Cloud users can be granted the following special permissions:
+
+* superset:admin
+* superset:alpha
+* superset:gamma (granted by default)
+
+These EDB Cloud permissions translate to Superset roles as follows:
+
+
+| User Superset Role  | Permissions Included in EDB Cloud User Role                                         | Superset Privileges                                                                    |
+| --------------------| ----------------------------------------------------------- | ----------------------------------------------------------------- |
+| Gamma               | No additional permissions needed |  View data that user has been granted access to, create dashboards          |
+| Alpha               | superset:alpha                                               | Gamma privileges, add or alter data sources                       |
+| Admin               | superset:admin_ permission                        | Alpha privileges, grant or revoke access to data to other users  |
```

Comment:
Hi DeeDee,

I think we can probably add what that particular role would not be able to do in abstract way. We're mentioning what is supported, but not "what is restricted". Do you think it would make sense, given we're already redirecting users to superset docs for more details.

For e.g. could we mention:  "Alpha cannot grant or revoke access from other users." or "Gamma users are not able to alter or add data sources."

Reply:
I think it is implied what they cannot do and that we don't need to add that information.

Reply:
I have to agree with @drothery-edb, there is no way to systematically enumerate what some thing X does not do, the list would be too large. Therefore, while mentioning what X can't do may be a part of some non formal tutorial, it is not appropriate to include it in official documentation.

Reply:
I'll add that I did not understand the "Superset Privileges" column until I read these comments. I agree that we probably don't need to add the negations, but it might be useful to be a bit more verbose, e.g., 

```suggestion
| Admin               | superset:admin_ permission                        | All Superset Alpha privileges, plus the ability to grant or revoke access to data to other users  |
```

Reply:
@drothery-edb : Also It should be just `superset:admin` not `superset:admin_ permission`

We've defined all these roles in Auth0 this way. If we attach `_permission` it would be misleading.